x-pack/plugins/security_solution/scripts/endpoint/endpoint_agent_runner/elastic_endpoint.ts

```diff
@@ -19,6 +24,8 @@ import {
   waitForHostToEnroll,
 } from '../common/fleet_services';
 import { getRuntimeServices } from './runtime';
+import type { PolicyData } from '../../../common/endpoint/types';
+import { ProtectionModes } from '../../../common/endpoint/types';
```

Comment:
nit: you can combine type imports with the other imports from the same module
```suggestion
import { ProtectionModes, type PolicyData } from '../../../common/endpoint/types';
```